content/docs/pipelines/reference/component-spec.md

```diff
@@ -121,51 +114,75 @@ use the component inside a pipeline.
 
 ## Using placeholders for command-line arguments
 
-### Consuming input by value (parameter)
+### Consuming input by value
 
-The placeholder is replaced by the value of the input argument:
+The `{inputValue: <input name>}` placeholder is replaced by the value of the input argument:
 
 * In `component.yaml`:
   
-  ```
+  ```yaml
   command: [program.py, --rounds, {inputValue: Rounds}]
   ```
 
 * In the pipeline code:
 
-  ```
+  ```python
   task1 = component1(rounds=150)
   ```
 
 * Resulting command-line code (showing the value of the input argument that
   has replaced the placeholder):
 
-  ```
+  ```shell
   program.py --rounds 150
   ```
 
-### Outputs
+### Consuming input by file
 
-Output paths are filled in by the pipeline system. The `outputPath` placeholder
-is replaced by a path. (The path can point to a mounted output volume, for
-example.) The parent directories of the path may or may not not exist. Your
-program must handle both cases without error.
+The `{inputPath: <Input name>}` placeholder is replaced by the (auto-generated) local file path where the system has put the argument data passed for the "<input name>" input.
```

Comment:
s/"<input name>"/"&lt;input name&gt;"

Otherwise the page attempts to render the input field -- see preview: https://deploy-preview-1277--competent-brattain-de2d6d.netlify.com/docs/pipelines/reference/component-spec/#consuming-input-by-value

Reply:
Fixed. Thanks for showing me the preview link.